### PR TITLE
Virtual scroll & Accessibility support

### LINKED ISSUE #407: 一堆问题
- [ ] cnpm 和 tnpm 没法 compile。

图片

- [ ] `npm run pub` 错误退出，没有发布 site，没有打 tag，没有 push。

图片

- [ ] 本地 ts 编辑器很多提示错误。

### PR DESCRIPTION
**Pull request by zombieJ**
# Preview 
https://rc-select-git-virtual.react-component.now.sh/

close #407

### CODE REVIEW

File: tests/__snapshots__/Tags.test.tsx.snap [hunk 1]
```diff
@@ -1,5 +1,203 @@
...
+                  </span>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```
Comment (afc163) [inline tests/__snapshots__/Tags.test.tsx.snap]: 看看能不能清掉一下无用的嵌套。

Comment (zombieJ) [reply tests/__snapshots__/Tags.test.tsx.snap]: Accessibility 和 Virtual list 要用，省不了……

**Comment by afc163** [comment]
https://github.com/react-component/select/issues/407

看看这个是不是都没了

**Comment by afc163** [comment]
https://rc-select-git-virtual.react-component.now.sh/?selectedKind=rc-select&selectedStory=single-animation&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fsource%2Fpanel

这个例子第一次展开没有动画。

**Comment by zombieJ** [comment]
嗯，动画改完 virtual list 再去升级 rc-trigger

**Pull request merged**